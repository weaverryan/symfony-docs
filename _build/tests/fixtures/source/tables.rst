
==========  ========================================  ==========================================
Route path  If the requested URL is /foo              If the requested URL is /foo/
==========  ========================================  ==========================================
/foo        It matches (200 status response)          It doesn't match (404 status response)
/foo/       It makes a 301 redirect to /foo/          It matches (200 status response)
==========  ========================================  ==========================================
