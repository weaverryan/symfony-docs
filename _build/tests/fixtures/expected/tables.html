<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<table class="table table-bordered">
    <thead>
    <tr>
        <th>Route path</th>
        <th>If the requested URL is /foo</th>
        <th>If the requested URL is /foo/</th>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td>/foo</td>
        <td>It matches (200 status response)</td>
        <td>It doesn't match (404 status response)</td>
    </tr>
    <tr>
        <td>/foo/</td>
        <td>It makes a 301 redirect to /foo/</td>
        <td>It matches (200 status response)</td>
    </tr>
    </tbody>
</table>
</body>
</html>